### PR TITLE
LinusLausberg/Exercise01

### PR DESCRIPTION
**Pull request by LinusL2005**

### CODE REVIEW

File: Exercises/exercises_01.py [hunk 1]
```diff
@@ -4,22 +4,42 @@
 # single print() which will give the answer(s) to that question.
 
 # what are the quotient and remainder after (integer) division of 2711274328912 by 23369245575
+print('Der Quotient von 2711274328912 dividiert durch 23369245575 ist:' , 2711274328912//23369245575 , 'Und der Rest ist:' , 2711274328912 % 23369245575)
```
Comment (HannahGedler) [inline Exercises/exercises_01.py:7]: ich fände es übersichtlicher, wenn du die Rechnung extra schreibst

File: Exercises/exercises_01.py [hunk 2]
```diff
@@ -4,22 +4,42 @@
...
-
+anzahl_auf = input_string.count('(')
+anzahl_zu = input_string.count(')')
```
Comment (atrejut) [inline Exercises/exercises_01.py:20]: I'd *very much* recommend keeping all code (variable names in this case) in english. You never know who'll read/edit/maintain that codebase later.

[Does not apply to the fixed strings, those clearly depend on your target audience -- or need to be prepared for i18n :)]

File: Exercises/exercises_01.py [hunk 3]
```diff
@@ -4,22 +4,42 @@
...
 # which and how many elements are common to the two lists above? 
-
-
-# how often does the character 'a' occur in each of the lists above?
\ No newline at end of file
+set_first_list = set(first_list)
+set_second_list = set(second_list)
+set_kombiniert = set_first_list.intersection(set_second_list)
+print('Hier sind alle Elemente aufgelistet, welche in beiden Reihungen vorkommen:'
+       , set_kombiniert ,'insgesamt sind es' ,  len(set_kombiniert) , 'Stück')
```
Comment (atrejut) [inline Exercises/exercises_01.py:34]: we didn't talk about codestyle yet, but the `,` at the beginning of a line seems particularly odd...

File: Exercises/exercises_01.py [hunk 4]
```diff
@@ -4,22 +4,42 @@
...
+
+# how often does the character 'a' occur in each of the lists above?
+anzahl_a_eins = 0
+anzahl_a_zwei = 0
+for x in first_list:
+      anzahl_a_eins += x.count('a')
```
Comment (atrejut) [inline Exercises/exercises_01.py:40]: better solutions (without any explicit `for`-loops) are possible.

File: Exercises/exercises_01.py [hunk 5]
```diff
@@ -4,22 +4,44 @@
 # single print() which will give the answer(s) to that question.
 
 # what are the quotient and remainder after (integer) division of 2711274328912 by 23369245575
+temp = 2711274328912//23369245575
+print('Der Quotient von 2711274328912 dividiert durch 23369245575 ist:' , temp , 'Und der Rest ist:' , 2711274328912 % 23369245575)
```
Comment (HannahGedler) [inline Exercises/exercises_01.py:8]: wenn du ein Leerzeichen zwischen den Doppelpunkt und die Variable schreibst kann man den Output besser lesen

**Pull request merged**